### PR TITLE
[MRG] Remove bad latin phrases so CI passes

### PR DESCRIPTION
**Pull request by sgibson91**
### Summary

This PR changes some latin phrases so that our tests pass.

### List of changes proposed in this PR (pull-request)

* e.g. --> for example
* i.e. --> that is

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: book/content/rdm/fairprinciples.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ should also regularly be updated as the research project progresses or diverge f
 Although started by a community operating in the life science, the FAIR principles have rapidly been adopted by
 publishers, funders, and pan-disciplinary infrastructure programmes and societies, in all disciplines. Many groups and
 organization are working to define guidance and tools to help researchers, as well as other stakeholders (librarians,
-funders, publishers, trainers etc.) to make data FAIR and assess its FAIRness level.
+funders, publishers, trainers et cetera) to make data FAIR and assess its FAIRness level.
```
Comment (jezcope) [inline book/content/rdm/fairprinciples.md]: Is *et cetera* necessarily any better than *etc*? Maybe "and so on", or better"for example/such as" at the start of the sentence, which flags up the non-exhaustive nature of the list up front?

Comment (sgibson91) [reply book/content/rdm/fairprinciples.md]: Actually, let me read the style guide 😂

Comment (sgibson91) [reply book/content/rdm/fairprinciples.md]: Updated in 356576b

**Comment by sgibson91** [comment]
Thank you @malvikasharan ! Why does travis hate all of my PRs though 😭

**Comment by malvikasharan** [comment]
There have been many merges in the last few days, that have missed Latin abbreviations. I am hunting them down today.

**Pull request merged**